FAQ Updates as per card #117 (#210)
casper-network/docs: source/docs/casper/faq/faq-developer.md

</details>

<details>
 <summary><b> When is the balance updated after a transaction?</b></summary>

 Execution occurs after consensus. As outlined [here in the dApp Development Guide]( https://casper.network/docs/dapp-dev-guide/deploying-contracts#check-deploy-status), deployments are queued in the system before being listed in a block for execution.

 Balance updates should occur after contract execution and block finalization.

</details>

<details>

>  <summary><b>How do I ensure sufficient balance during multiple transfers?</b></summary>

Reviewer: ```suggestion
 <summary><b>How do I handle a transaction composed of multiple transfers?</b></summary>
```

Reviewer: I feel that the answer does not match the question. I would update the question in this case.

>  Applying a unique ID to each transfer can mitigate issues with multiple transfers. Once deployed to a block, the transaction effectively finalizes.

Reviewer: ```suggestion
 Applying a unique ID to each transfer can mitigate issues with multiple transfers. Once deployed to a block, the network finalizes the transaction composed of multiple transfers.
```

</details>

> ### Invalid Purse Errors {#invalid-purse-errors}

Reviewer: I suggest creating one section called "Errors" and combining the Invalid Purse Error and the API Error underneath it.

<details>
 <summary><b>What factors cause the 'APIError::InvalidPurse' error?</b></summary>

 The three main factors that cause an InvalidPurse error are:

 1. The purse in question does not exist.

>  2. The purse does not of the U512 type.

Reviewer: ```suggestion
 2. The purse is not of type U512.
```
